app/app/settings.py

```diff
@@ -74,12 +74,21 @@
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.auth.middleware.RemoteUserMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'social_django.middleware.SocialAuthExceptionMiddleware',
     'applicationinsights.django.ApplicationInsightsMiddleware',
+    'custom_auth.CustomSSOMiddleware', 
 ]
 
+# added for custom SSO 
+CREATE_USER = env.bool("CREATE_USER", default=True)
+IS_SUPER = env.bool("IS_SUPER", default=False)
```

Comment:
Given that we have a way of controling admin status via the headers now, I'd remove this general flag. I don't see a scenario where we'd want everyone who logs in via a header to be an admin where we couldn't simply update the authentication proxy to set the correct header value.

Reply:
Remove both ```IS_SUPER``` and ```CREATE_USER```?  The earlier comment suggests otherwise _"If we set the is-admin  flag via the environment, then both UserA and UserB would be logged in as admin since X-AUTH-PROXY-GROUPS is ignored."_

I can remove it but like to confirmation.

Reply:
That comment mentioned an undesirable state. Yes, do remove the flag.